src/CMakeLists.txt

```diff
@@ -289,6 +289,15 @@ include_directories(SYSTEM ${Exiv2_INCLUDE_DIRS})
 list(APPEND LIBS ${Exiv2_LIBRARIES})
 add_definitions(${Exiv2_DEFINITIONS})
 
+if(USE_FFTW3)
+  find_package(FFTW3)
+  if(FFTW3_FOUND)
+    include_directories(SYSTEM ${FFTW3_INCLUDE_DIRS})
```

Comment:
We really need to migrate to modern cmake with imported targers.
This is fscking horrible.